hubspot/connection/exc.py

```diff
@@ -30,7 +30,8 @@ class HubspotClientError(HubspotException):
     of 40X, except 401
 
     :param unicode request_id:
-    :param dict optional error_data:
+    :param unicode error_message: The error message returned by HubSpot
```

Comment:
Sorry for being pedantic but the order is wrong, `error_message` should be before `request_id`